cmd/dump.go

```diff
@@ -185,35 +189,41 @@ func runDump(ctx *cli.Context) error {
 		}
 	}
 
-	tmpDir := ctx.String("tempdir")
-	if _, err := os.Stat(tmpDir); os.IsNotExist(err) {
-		fatal("Path does not exist: %s", tmpDir)
-	}
+	if ctx.Bool("skip-db") {
+		// Ensure that we don't dump the database file that may reside in setting.AppDataPath or elsewhere.
+		dumper.GlobalExcludeAbsPath(setting.Database.Path)
```

Comment:
Is it safe to assume that `setting.Database.Path` is an absolute path?

Reply:
I seen other areas that perform a `ctx.IsSet()` before a `ctx.Bool()`, but after reading the docs associated to the Bool() method it should return false if `--skip-db` is not provided; so I decided to only use this function. Please let me know if this is a problem

Reply:
But the path maybe empty? Whether it will affect the result?

Reply:
From looking at `GlobalExcludeAbsPath()` (see [here](https://github.com/go-gitea/gitea/blob/b06aac40e6552b0ce1f7b8a92c977fcc27566f68/modules/dump/dumper.go#L104)), if it was passed the empty string, it would essentially append "." to `dumper.globalExcludeAbsPaths`.

Even though it's not an absolute path, it doesn't look like it would affect the result as we if we add a "." to this slice any subsequent paths that are just the empty string would also be cleaned to be "." and therefore be excluded since that string is in the slice. If you feel differently, please let me know and I'll add an empty string check for this